mne/io/ctf/info.py

```diff
@@ -31,11 +37,18 @@ def _pick_isotrak_and_hpi_coils(res4, coils, t):
     n_coil_head = 0
     for p in coils:
         if p['valid']:
+            if p['kind'] in [CTF.CTFV_COIL_LPA, CTF.CTFV_COIL_RPA,
+                             CTF.CTFV_COIL_NAS]:
+                kind = FIFF.FIFFV_POINT_CARDINAL
+                ident = _ctf_to_fiff[p['kind']]
+            else:  # CTF.CTFV_COIL_SPARE
+                kind = FIFF.FIFFV_POINT_HPI
```

Comment:
Why does this map to `HPI` now, when in the dict above (and based on the name) it would be `EXTRA`?